gh-71385: add relative delta parameter for assertAlmostEqual and assertNotAlmostEqual (#96881)
python/cpython: Misc/NEWS.d/next/Library/2022-09-16-21-51-41.gh-issue-71385.r5XUzt.rst

> A new keyword "rel_delta" was added to the assertAlmostEqual and assertNotAlmostEqual methods of the unittest.TestCase class. It enables to compare two numbers and objects supporting addition, subtraction and scalar number multiplication to be compared with a maximum relative difference to each other.

Reviewer: ```suggestion
A new parameter ``rel_delta`` was added to the :meth:`unittest.TestCase.assertAlmostEqual` and :meth:`unittest.TestCase.assertNotAlmostEqual`. It allows comparing two objects with a maximum relative difference to each other.
```
